packages/contractkit/src/wrappers/Attestations.ts

```diff
@@ -178,14 +178,14 @@ export class AttestationsWrapper extends BaseWrapper<Attestations> {
     )
 
     const isIncomplete = (status: AttestationState) => status === AttestationState.Incomplete
-    const hasNotExpired = (time: number) => nowInUnixSeconds < time + expirySeconds
+    const hasNotExpired = (time: number) => currentBlockNumber < time + expirySeconds
```

Comment:
This looks incorrect, you're comparing a block number to a time plus a block number.  Would it be better to use `isAttestationExpired`?

Reply:
The variable is called `time`, but it is actually a block number, so renamed it to clarify. I wanted to avoid using `isAttestationExpired` to reduce the number of view calls necessary. I talked with @jmrossy and we'll probably have a more optimized view call for this anyways